app/src/main/java/org/mozilla/focus/utils/Settings.java

```diff
@@ -37,11 +37,9 @@ private Settings(Context context) {
     }
 
     public boolean shouldBlockImages() {
-        // Not shipping in v1 (#188)
-        /* return preferences.getBoolean(
+        return preferences.getBoolean(
                 resources.getString(R.string.pref_key_performance_block_images),
-                false); */
-        return false;
+                true);
```

Comment:
move true to a private static final Sting BLOCK_IMAGE_DEFAULT would be a good idea.